src/main/java/com/alexx/employeesAndDepartments/controllers/DepartmentController.java

```diff
@@ -25,12 +27,14 @@ public DepartmentController(DepartmentService departmentService) {
     @PostMapping("/departments")
     @Operation(summary = "Add department", description = "Add new department, Request body with name of department required, ID is generated automatically")
     public ResponseEntity<DepartmentDTO> addDepartment(@RequestBody DepartmentDTO departmentDTO) {
+        log.info("Add department");
```

Comment:
So if there is a problem with adding a department, who could we trace it in logs if you for every addDepartment put the same message? If there are millions of requests and you need to trace one request that failes

Reply:
I don't know if it's possible to store personal information (name, etc.) in the log